docs/telemetry.md

```diff
@@ -0,0 +1,51 @@
+Telemetry
+========================
+
+Telemetry is a feature that helps us to remotely collect user experience data to make our product better.
+To achieve this it creates, collects and sends the business events to remote server for further analyzing.
+If remote server is not accessible the collector stores all the events in the local DB until the remote server be reachable.
+For now is only the task commands execution events are implemented. It collects all the tasks and sends to the data lake remote server.
+
+## Config
+All the configuration related to telemetry is place in `config/packages/telemetry.yaml`
+```yaml
+parameters:
+  telemetry_enabled: "%env(default:telemetry_enabled_default:bool:TELEMETRY_ENABLED)%"
+  telemetry_enabled_default: true
+  telemetry_data_lake_endpoint_url: "%env(string:TELEMETRY_DATA_LAKE_ENDPOINT_URL)%"
```

Comment:
Datalake is only one of the possible servers and can be replaced in the future. Please, rename config to the `telemetry_server_url` and introduce one more param `telemetry_transport` which will be `datalake` by default. Also, the strategy must be implemented to call correct sender based on transport data.